spark-bigquery-dsv2/spark-bigquery-dsv2-common/pom.xml

```diff
@@ -27,6 +27,11 @@
             <artifactId>spark-bigquery-connector-common</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.openlineage</groupId>
+            <artifactId>spark-extension-interfaces</artifactId>
+            <version>1.19.0</version>
```

Comment:
The version should only appear in the spark-bigquery-parent POM file, as part of the dependencyManagement

Reply:
Done